index.js

```diff
@@ -53,6 +53,7 @@ module.exports = {
     'mocha/no-pending-tests': 'error',
     'mocha/no-skipped-tests': 'warn',
     'mocha/valid-test-description': 'error',
+    'no-template-curly-in-string': 'off',
```

Comment:
bunsen itself doesn't have a `eslint-plugin-<bunsen>` that is listed as a dependency. I believe this repo serves as a place to hold all our eslint rules. So our `.eslintrc` is simply
```
{
  "extends": ["frost-standard"],
}
```